scripts/modules/commands.js

```diff
@@ -132,13 +145,709 @@ import {
+						console.error(chalk.red(`\n  ${errorMsg}\n`));
+						return;
+					}
+				}
+			} catch {
+				// Continue polling on error
+			}
+		}
+
+		// Check if we timed out while still generating
+		if (isStillGenerating(briefStatus)) {
+			spinner.warn('Task generation is still in progress');
+			console.log('');
+			console.log(
+				chalk.yellow('  Tasks are still being generated in the background.')
+			);
+			console.log(chalk.white('  Check the brief URL above for progress.'));
+		} else {
+			spinner.succeed(
+				taskCount > 0
+					? `Done! ${taskCount} tasks generated`
+					: 'Task generation complete'
+			);
+		}
+		console.log('');
+
+		// Show invite URL for adding more teammates later (orgSlug already extracted above)
+		if (orgSlug) {
+			const urlParts = result.brief.url.match(/^(https?:\/\/[^/]+)/);
+			const baseUrl = urlParts ? urlParts[1] : '';
+			const membersUrl = `${baseUrl}/home/${orgSlug}/members`;
+			const clickableMembersUrl = `\x1b]8;;${membersUrl}\x07${chalk.cyan.underline(membersUrl)}\x1b]8;;\x07`;
+			console.log(
+				chalk.gray('  Invite more teammates: ') + clickableMembersUrl
+			);
+			console.log('');
+		}
+
+		// Set context to the new brief using resolveBrief (same as tm context <url>)
+		try {
+			const brief = await tmCore.tasks.resolveBrief(result.brief.url);
+			const briefName =
+				brief.document?.title || `Brief ${brief.id.slice(0, 8)}`;
+
+			// Get org info for complete context
+			let orgName;
+			try {
+				const org = await authManager.getOrganization(brief.accountId);
+				orgName = org?.name;
+			} catch {
+				// Non-fatal if org lookup fails
+			}
+
+			await authManager.updateContext({
+				orgId: brief.accountId,
+				orgName,
+				orgSlug,
+				briefId: brief.id,
+				briefName,
+				briefStatus: brief.status,
+				briefUpdatedAt: brief.updatedAt
+			});
+
+			console.log(
+				chalk.green('  ✓ ') +
+					chalk.white('Context set! Run ') +
+					chalk.cyan('tm list') +
+					chalk.white(' to see your tasks.')
+			);
+		} catch (contextError) {
+			// Log the actual error for debugging
+			log('debug', `Context auto-set failed: ${contextError.message}`);
+			console.log(
+				chalk.yellow('  Could not auto-set context. Run ') +
+					chalk.cyan(`tm context ${result.brief.url}`) +
+					chalk.yellow(' to connect.')
+			);
+		}
+		console.log('');
+	} catch (error) {
+		if (spinner?.isSpinning) spinner.fail('Failed');
+		console.error(chalk.red(`\n  Error: ${error.message}\n`));
+	}
+}
+
+/**
+ * Helper to create aligned command entries
+ */
+function createCommandEntry(command, description, indent = '  ') {
+	const cmdColumn = 47; // Fixed column width for commands
+	const paddingNeeded = Math.max(1, cmdColumn - indent.length - command.length);
+	return (
+		chalk.cyan(indent + command) +
+		' '.repeat(paddingNeeded) +
+		chalk.gray(description)
+	);
+}
+
+/**
+ * Display Hamster-specific help (simplified command list)
+ */
+function displayHamsterHelp() {
```

Comment:
please make sure you're adding any new functions into a typescript file somewhere and calling it here instead of creating the function inside the commands.js file we're trying to refactor and move away from.